app/assets/stylesheets/admin/global/_base.sass

```diff
@@ -16,6 +21,18 @@ div.container
   span
     @extend %visually-hidden
 
+thead, h1, h2, h3, h4, h5, h6
+  color: $heading-color
+
+thead, h1, h2, h3
+  font-family: $heading-fonts
+
+h1, h2, h3, h4, h5, h6
```

Comment:
Cool man. I have removed the link from my comment as it's a link to a non-public repo.